app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java

```diff
@@ -350,6 +355,9 @@ private void maybeRemoveUserPicture() {
     }
 
     private void updateUserPicture() {
+        if (!getGoogleApiClient().hasConnectedApi(Plus.API)) {
```

Comment:
I am not sure if this is the only place. How can we make sure that the other parts are ok too. I think we may have problems with other GMS calls we are doing.